config/certmanager/kustomization.yaml

```diff
@@ -5,3 +5,6 @@ resources:
 
 configurations:
 - kustomizeconfig.yaml
+
+namespace: kong-system
```

Comment:
Do we actually need this?

The `default` kustomization defines the `kong-system` namespace in https://github.com/Kong/kong-operator/blob/37604cae816b87f8ab23693f3c37d99d1b84a59f/config/default/kustomization.yaml#L4 so it should override it here, right?